src/App.js

```diff
@@ -1,28 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
```

Comment:
- [ ] Kindly import `React` in this component. This is because JSX gets internally into many React. createElement() function calls and each of them returns an object. Now because of this, we need to import React from “react” since internally every JSX is creating a React Component using JSX transformer. You can read more about this [here](https://www.geeksforgeeks.org/why-do-you-need-to-import-react-in-functional-components/#:~:text=Explanations%3A%20The%20JSX%20gets%20internally,React%20Component%20using%20JSX%20transformer.)

- [ ] The `App.js` component should be class-based. Kindly make use of a class component as it is part of the project requirement. See suggestion below;

```js
class App extends React.Component {
  render() {
    return {
        <div className="App">
          <CalculatorBody />
       </div>
    };
  }
}

Reply:
Thanks a lot @julie-ify for drawing my attention to these. I have resolved them.